web/src/app/projects/view/view.component.html

```diff
@@ -49,6 +49,12 @@
         <div class="project-body__header__content__item__pings" fxLayout="row" fxLayoutGap="5px">
           <mat-icon svgIcon="ping_icon"></mat-icon><span>{{ project.getTotalPings() }} | Total Pings</span>
         </div>
+        <div *ngIf= "!isFollower" (click)="incrementFollowers()" class="project-body__header__content__item" fxLayout="row" fxLayoutGap="5px">
```

Comment:
I think the function name is confusing. The user is following the project, it does increment but that is not their action. I think `followProject` and `unfollowProject` would make more sense here

Reply:
Cool